contrib/completions/bash/kubectl

```diff
@@ -135,6 +166,8 @@ __handle_noun()
 
     if __contains_word "${words[c]}" "${must_have_one_noun[@]}"; then
         must_have_one_noun=()
+    elif __contains_word "${words[c]%!s(MISSING)}" "${must_have_one_noun[@]}"; then
```

Comment:
Bad escaping of bash code in golang string in fact: https://github.com/spf13/cobra/pull/255/files#diff-3db5cab63dbb32fff4264f5449fd2904R192